### PR TITLE
Drop ignored models and ignored or unsupported fields from the dmmf

### PR DESCRIPTION
**Pull request by do4gr**
add testing for rendering parsing for dmmf, ast and dml

already spoke to Tim that the filtering is fine for him. Will also check with Sid.

### CODE REVIEW

File: query-engine/query-engine/src/opt.rs [hunk 1]
```diff
@@ -11,11 +11,11 @@ pub enum Subcommand {
...
-#[derive(Debug, Clone, StructOpt)]
-pub struct DmmfToDmlInput {
-    #[structopt(name = "path")]
-    pub path: String,
-}
+// #[derive(Debug, Clone, StructOpt)]
+// pub struct DmmfToDmlInput {
+//     #[structopt(name = "path")]
+//     pub path: String,
+// }
```
Comment (tomhoule) [inline query-engine/query-engine/src/opt.rs:18]: Can this be deleted?

File: libs/datamodel/core/src/json/dmmf/to_dmmf.rs [hunk 2]
```diff
@@ -57,7 +57,11 @@ fn model_to_dmmf(model: &dml::Model) -> Model {
         name: model.name.clone(),
         db_name: model.database_name.clone(),
         is_embedded: model.is_embedded,
-        fields: model.fields().map(|f| field_to_dmmf(model, f)).collect(),
+        fields: model
+            .fields()
+            .filter(|field| !field.is_ignored() && !matches!(field.field_type(), FieldType::Unsupported(_)))
+            .map(|f| field_to_dmmf(model, f))
+            .collect(),
```
Comment (tomhoule) [inline libs/datamodel/core/src/json/dmmf/to_dmmf.rs:64]: Oh that's what this module is for, I understand now.

**Review by tomhoule — APPROVED** [review]
lgtm

File: query-engine/query-engine/src/opt.rs (hunk 1 above)
Comment (do4gr) [reply query-engine/query-engine/src/opt.rs:18]: Yes, I commented it out to see whether anything breaks and also confirmed with Tim that we do no longer go from dmmf-> dml

**Review by dpetrick — APPROVED** [review]

**Pull request merged**